templates/terraform/examples/app_engine_application_url_dispatch_rules_basic_example.tf.erb

```diff
@@ -0,0 +1,43 @@
+resource "google_storage_bucket" "bucket" {
+	name = "<%= ctx[:vars]['bucket_name'] %>"
```

Comment:
You'll need to supply a bucket_name variable in the example definition above.

Reply:
provided in terraform.yaml